portfolio/src/main/webapp/map.js

```diff
@@ -0,0 +1,90 @@
+class MapHandler {
+  // The coordinates for all markers and their related properties.
+  static MAP_CENTER = {
```

Comment:
All constants should have JSDocs. Also, seems like the coords could be broken up into their own Coordinates Class that can be instantiated with a lat/long/title/id.

Reply:
Goog idea! Thanks :)